src/plugins/intel_cpu/tests/functional/single_layer_tests/shape_ops.cpp

```diff
@@ -84,10 +87,22 @@ class ShapeOpsCPUTest : public testing::WithParamInterface<shapeOpsParams>, virt
             const auto& funcInput = funcInputs[i];
             ov::runtime::Tensor tensor;
             if (i == 1) {
-                tensor = ov::runtime::Tensor{ov::element::i32, targetInputStaticShapes[i]};
-                auto inputData = tensor.data<ov::element_type_traits<ov::element::i32>::value_type>();
-                for (size_t j = 0lu; j < data[idx].size(); ++j) {
-                    inputData[j] =  data[idx][j];
+#define RESHAPE_TEST_CASE(INT_TYPE) \
+        case ov::element::Type_t::INT_TYPE: { \
+                    tensor = ov::runtime::Tensor{ov::element::INT_TYPE, targetInputStaticShapes[i]}; \
+                    auto inputData = tensor.data<ov::element_type_traits<ov::element::INT_TYPE>::value_type>(); \
+                    for (size_t j = 0lu; j < data[idx].size(); ++j) { \
+                            inputData[j] =  data[idx][j]; \
+                    } \
+                    break; \
+             }
+                switch (secondInPrc) {
+                    RESHAPE_TEST_CASE(i64)
+                    RESHAPE_TEST_CASE(i32)
+                    RESHAPE_TEST_CASE(i8)
```

Comment:
`i8` doesn't make sense here. The second input is a parameter of the operation that is not quantized. Testing `i8` on the second input may only increase the amount of test cases without a practical value.